packages/core/src/BlockNoteEditor.ts

```diff
@@ -137,6 +141,11 @@ export class BlockNoteEditor {
         options.onEditorReady?.(this);
         options.initialContent &&
           this.replaceBlocks(this.topLevelBlocks, options.initialContent);
+        document.addEventListener(
```

Comment:
I'm not entirely sure we want this here. Can't we keep this in the DraggableBlocksPlugin?

Reply:
We could, and then ig we would pass the cursor pos as a prop to the menu. I think that if we want to allow consumers to check the currently mouse-hovered block outside the DraggableBlocksPlugin though, we would still need smth like this. Imo it's better to init the listener in BlockNoteEditor and pass the mouse coords down to DraggableBlocksPlugin, rather than relying on DraggableBlocksPlugin to update mouse coords. Wdyt?

Reply:
I think there is a use case for getting the block hovered by the mouse outside the DraggableBlocksPlugin btw, e.g. a slash menu command that involves 2 blocks, where you can select the 2nd block with the mouse.